packages/ui/origin-ui-core/src/components/certificates/ExchangeInboxPage.tsx

```diff
@@ -128,16 +128,19 @@ export function ExchangeInboxPage(): JSX.Element {
                                     type="number"
                                     value={price}
                                     onChange={(ev) => {
-                                        const newValue = parseFloat(ev.target.value);
-                                        if (!isNaN(newValue)) setPrice(newValue);
+                                        const newValue = parseFloat(
+                                            ev.target.value.replace(',', '.')
+                                        );
+                                        if (!isNaN(newValue))
+                                            setPrice(Math.floor(newValue * 100) / 100);
```

Comment:
Currently I am unable to write a number with dot, only numbers with comma could be typed into this input. 
Please, check.